### PR TITLE
Game testing

### PR DESCRIPTION
**Pull request by MarvellousUbani**

### CODE REVIEW

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey, @KerronKing @MarvellousUbani Great!!! work with Testing 

image. 

However there are some testing missing.

- [x]  You should write a test for the player class, so you can make `player_spec` file.
- [x] There no test case for a game draw

[Optional] Not required 
You can add --format documentation to your `.rspec `. This gives a better display message from tests.

**Comment by bolah2009** [comment]
## Currently under review

File: spec/board_spec.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      board.grid[0] = ['', '', 'X']
+      board.grid[1] = ['', 'X', '']
+      board.grid[2] = ['X', '', '']
+      expect(board.check_win('X')).to eql(true)
+    end
+    it 'returns false if board is full and no win is possible (drawn game)' do
+      board.grid[0] = %w[X O O]
+      board.grid[1] = %w[O X X]
+      board.grid[2] = %w[X X O]
+      expect(board.check_win('X')).not_to eq(true)
```
Comment (samgaco) [inline spec/board_spec.rb:51]: Testing symbol2 might be a good idea as well

**Review by samgaco — APPROVED** [review]
Hi @KerronKing and @MarvellousUbani :high_brightness: 

All the requirements have been covered, hence the project is approved, you can merge and submit the project completion form. Good luck with rails :100: 

----------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**